pallets/ajuna-awesome-avatars/src/types/avatar/versions/v2/avatar_utils.rs

```diff
@@ -1064,6 +1064,10 @@ where
 		(avatar.dna[21..32]).copy_from_slice(&value);
 	}
 
+	/// Checks if the `array_1` can be upgraded using `array_2`
```

Comment:
Maybe we can find some good naming here to make it clear. I know it is tedious, but I think it is worth it. How about something like:

* `upgradable` - `input`
* I have heard the term sacrifice, I think it could fit here too: `upgradable` - `sacrifice`?
* We talk about a progress match, so maybe we could also use `progressible`?

Any other ideas?

Reply:
I remember now that you use the term `leader` and `sacrifice`? Maybe we can use these terms here too? I know that the functions are more generic, but if they are exclusively used in this context, I think it makes sense to use the domain names.

Reply:
Yeah, makes sense to adjust it like that